src/main/java/io/aiven/klaw/config/SecurityConfigNoSSO.java

```diff
@@ -87,35 +90,38 @@ private void shutdownApp() {
 
   @Override
   protected void configure(HttpSecurity http) throws Exception {
-
-    String[] staticResources = {
-      "/lib/**",
```

Comment:
We need these libs, js and assets for angular js to work. If it's not conflicting with coral, we can add them back.